include/libremidi/backends/winmm/midi_in.hpp

```diff
@@ -64,7 +64,7 @@ class midi_in_winmm final
   bool do_open(unsigned int portNumber)
   {
     MMRESULT result = midiInOpen(
-        &this->inHandle, portNumber, (DWORD_PTR)&midiInputCallback, (DWORD_PTR)this,
+        &this->inHandle, portNumber, reinterpret_cast<DWORD_PTR>(&midiInputCallback), reinterpret_cast<DWORD_PTR>(this),
```

Comment:
do these work as static_cast instead of reinterpret_cast?

Reply:
These ones don't as its converting from pointers to integers and C++ requires a reinterpret cast for that, although a std::bit_cast would also work in a similar way that a static cast would.

Reply:
Changed these to std::bit_cast.